tools/commander/commander.go

```diff
@@ -0,0 +1,125 @@
+package commander
+
+import (
+	"io"
+	"os"
+	"os/exec"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/bitrise-io/go-utils/log"
+)
+
+// Command controls the command run.
+type Command struct {
```

Comment:
It might can be `Process` as well. As the process can have these fields, not like a Command has cmd field.